vlinder/src/app/register/[accessToken]/@step_tree/page.tsx

```diff
@@ -52,7 +52,7 @@ export default function Page() {
           <Button
             color={sexualOrientation === o ? "primary" : "default"}
             className="w-full mb-4"
-            data-sexualOrientation={o}
+            data-sexualorientation={o}
```

Comment:
Sexual orientation is heterosexual or asexual not male/female/other (slechte review kobe want staat al in development)